documentation/breaking-changes/upcoming-breaking-changes.md

```diff
@@ -27,3 +27,33 @@
 
 # Upcoming Breaking Changes
 
+    ## Release 13.0.0 - November 2024
+
+    The following cmdlets were affected this release:
+
+    **All Cmdlets**
+    - All cmdlets with SystemData properties have their property-type switched from struct to string.
```

Comment:
Please refer to https://github.com/Azure/azure-powershell/blob/3dfb95de6647a49c2ed584abe18195f3975dc33e/tools/StaticAnalysis/Exceptions/Az.DesktopVirtualization/BreakingChangeIssues.csv to preannounce all the non-v4-upgrading breaking changes